### PR TITLE
Create reservations

### PR DESCRIPTION
**Pull request by SilGosker**

### CODE REVIEW

File: Kbs.Business/Reservation/BoatTypeTEMPORARYIFNEEDEDMAKEANEWONEPLSTHISISFORMYSANITYANDTESTINGIFIAMCOMPETENDliefsjonathan.cs
Comment (Katanax28) [inline Kbs.Business/Reservation/BoatTypeTEMPORARYIFNEEDEDMAKEANEWONEPLSTHISISFORMYSANITYANDTESTINGIFIAMCOMPETENDliefsjonathan.cs:1]: delete this please

File: Kbs.Wpf/Reservation/CreateReservation/SelectLength/SelectLength.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        Tuple<ReservationTime, BoatEntity> chosenTimeAndBoat;
+        public int lenghtSelected;
+        public DateTime selectedStartTime = new DateTime();
+        public SelectLength(INavigationManager navigationManager, Tuple<ReservationTime, BoatEntity> chosenTimeAndBoat)
+            {
+            _navigationManager = navigationManager;
+            this.chosenTimeAndBoat = chosenTimeAndBoat;
+            lenghtSelected = 30;
+            InitializeComponent();
+            ViewModel.MakeSelectLengthViewModelGreatAgain(MakeComboboxAvailableTimes(),chosenTimeAndBoat.Item2.Name,chosenTimeAndBoat.Item1.StartTime);
```
Comment (Katanax28) [inline Kbs.Wpf/Reservation/CreateReservation/SelectLength/SelectLength.xaml.cs:49]: interesting name

File: Kbs.Business/Kbs.Business.csproj [hunk 2]
```diff
@@ -6,6 +6,12 @@
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Reservation\BoatTypeTEMPORARYIFNEEDEDMAKEANEWONEPLSTHISISFORMYSANITYANDTESTINGIFIAMCOMPETENDliefsjonathan.cs" />
+    <Compile Remove="Reservation\DateEntity.cs" />
+    <Compile Remove="Reservation\ReservationTimeFactory.cs" />
+  </ItemGroup>
```
Comment (SilGosker) [inline Kbs.Business/Kbs.Business.csproj:13]: Remove pls❤️

File: Kbs.Wpf/Kbs.Wpf.csproj [hunk 3]
```diff
@@ -8,6 +8,18 @@
...
+    <Compile Remove="Reservation\CreateReservation\SelectLength\SelectLengthLengthOptionViewModel.cs" />
+    <Compile Remove="Reservation\CreateReservation\SelectLength\SelectLengthPage.xaml.cs" />
+    <Compile Remove="Reservation\CreateReservation\SelectTime\SelectTimeDayOfWeek.cs" />
+    <Compile Remove="Reservation\CreateReservation\SelectTime\SelectTimePossibleTimeViewModel.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Page Remove="Reservation\CreateReservation\SelectLength\SelectLengthPage.xaml" />
+  </ItemGroup>
+
```
Comment (SilGosker) [inline Kbs.Wpf/Kbs.Wpf.csproj:22]: Remove pls ❤️

**Pull request merged**